math/mobius/mobius.go

```diff
@@ -0,0 +1,44 @@
+// mobius.go
+// description: Returns μ(n)
+// details:
+// For any positive integer n, define μ(n) as the sum of the primitive nth roots of unity.
+// It has values in {−1, 0, 1} depending on the factorization of n into prime factors:
+//   μ(n) = +1 if n is a square-free positive integer with an even number of prime factors.
+//   μ(n) = −1 if n is a square-free positive integer with an odd number of prime factors.
+//   μ(n) = 0 if n has a squared prime factor.
+// wikipedia: https://en.wikipedia.org/wiki/M%C3%B6bius_function
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see mobius_test.go
+
+package mobius
+
+import (
+	p "github.com/TheAlgorithms/Go/math/prime"
+)
+
+// Mu is the Mobius function
+// This function returns μ(n) for given number
+func Mu(n int) int {
+
```

Comment:
You have quite a few newlines when they aren't necessary. For example
```Go
func Mu(n int) int {
	if n <= 1 {
		return 1
	}

 	var primeFactorCount int
 	for i := 1; i <= n; i++ {
		...
```
This is mostly preference and I don't mind if you keep it as is but as a reader, I personally find it easier to work through a codebase with good logical blocks that are separated nicely.